Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -1309,21 +1323,28 @@ private extension ChatMessage {
 
         if let currentUser = context.currentUser {
             isSentByCurrentUser = currentUser.user.id == dto.user.id
-            currentUserReactions = Set(
-                MessageReactionDTO
-                    .loadReactions(ids: dto.ownReactions, context: context)
-                    .compactMap { try? $0.asModel() }
-            )
+            if !dto.ownReactions.isEmpty {
+                currentUserReactions = Set(
+                    MessageReactionDTO
+                        .loadReactions(ids: dto.ownReactions, context: context)
+                        .compactMap { try? $0.asModel() }
+                )
+            } else {
+                currentUserReactions = []
+            }
```

Comment:
Why was this introduced? 🤔

Reply:
Skipping the fetch request when no ownReaction ids are present